src/pnconnector/src/util/resource.go

```diff
@@ -92,7 +92,7 @@ func ResourcesEventsYaml() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "../resources/events.yaml", size: 39644, mode: os.FileMode(493), modTime: time.Unix(1655975141, 0)}
+	info := bindataFileInfo{name: "../resources/events.yaml", size: 39644, mode: os.FileMode(493), modTime: time.Unix(1660886285, 0)}
```

Comment:
This change may be included to this pull request becauase resource.go is automatically generated. Does it make sense to include this change?